typescript/src/bitcoin.ts

```diff
@@ -779,3 +780,60 @@ export function isP2WSHScript(script: Buffer): boolean {
     return false
   }
 }
+
+/**
+ * Represents a structured JSON format for a Bitcoin transaction. It includes
+ * detailed information about its inputs and outputs, as well as the transaction
+ * itself.
+ */
+interface TxJSON {
+  hash: string
+  version: number
+  locktime: number
+  inputs: {
+    hash: string
+    index: number
+    sequence: number
+    script: string
+    witness: string[]
+  }[]
+  outputs: {
+    value: number
+    script: string
+    address: string
+  }[]
+}
+
+/**
+ * Converts a raw Bitcoin transaction into a structured JSON format.
+ * @param rawTransaction - A raw Bitcoin transaction in hexadecimal string format.
+ * @param bitcoinNetwork - Bitcoin network.
+ * @returns A structured JSON object representing the transaction.
+ */
+export function txToJSON(
```

Comment:
This is used for unit tests only right? If so, I would put it under `test/utils` to not pollute the codebase with test-only code. Same for `TxJSON`.

Reply:
Done in 881e19de9fc41da0d3f8eaa0c5fb479870c49c04.